SConstruct

```diff
@@ -34,7 +34,7 @@ def get_version():
         return version_file.read().rstrip()
 
 API_VERSION_MAJOR = "1"
-API_VERSION_MINOR = "6"
+API_VERSION_MINOR = "7"
```

Comment:
This patch actually break API, and daos_oclass_type_t has been redefined to u32, and eg OC_SX has been changed to overflow u16 compared before, so IOR vol-daos need be rebuilt, otherwise you will see DER_INVAL returned by testing which is mostly of abvoe 26 test failure reasons i think, i manually rebuilt locally and confirm it rebuilt help fix issue.

Reply:
This patch actually break API, and daos_oclass_type_t has been redefined to u32, and eg OC_SX has been changed to overflow u16 compared before, so IOR vol-daos need be rebuilt, otherwise you will see DER_INVAL returned by testing which is mostly of abvoe 26 test failure reasons i think, i manually rebuilt locally and confirm it rebuilt help fix issue.

Reply:
And i think it better to bump API_VERSION_MAJOR to 2.

Reply:
That is what is **need**ed, not just disireable.  If we do not bump the major version, then DNF does not have accurate enough information to install the version/build of daos client consumers (tests, middleware, etc.) that have the correct (i.e. matching)  ABI.